### ISSUE TITLE
Cancel a sale

### ISSUE THREAD

**Issue by AlexandrePalo** [borgia-app/Borgia#10]
# Cancel a sale

Explore solution to cancel a sale: update stock, balances, ...
Permissions to do that ? What for ?

**Comment by Eyap53** [comment]
I don't think this is a good idea to delete a sale already done. **Instead, an admin can use an "exceptional movement" to refund the user.**

### Another possibility : 
When an operator makes a sale in a shop, the infos about it are displayed just after.
We may add the possibility to "put on hold" the sale in this screen, and add a "Cancel" button.
- Cancel button will return to the sale form
- Continue button will apply the sale.

However, I'm not sure if it's really a pertinent feature. What do you think ?

**Comment by rowsky** [comment]
Let's reopen the issue !
It is now needed by Siber's too !

I don't think there is a need to put on hold the sale. I think that we might be able to cancel any sale.
Also canceling a sale does not mean deleting it, we can add a status that should be "done" for every existing sale and assign it to cancelled when needed. Canceling a sale will refund the user and may have an impact on stock inventory.
Ofc only admins (maybe treasurers) should be able to do this.
